tiny changes for next

matthias-luger opened the pull request:
using next to implement SSR

Flou21 committed:
next start

matthias-luger committed:
implemented next

matthias-luger committed:
escape document

matthias-luger committed:
finished styling and some functionality
site should now behaive like before

matthias-luger committed:
finished v1

matthias-luger committed:
Merge branch 'next' of https://github.com/Coflnet/hypixel-react into next

matthias-luger committed:
fixed colour

matthias-luger committed:
fixed flipper width element

matthias-luger committed:
fixed startpage images

matthias-luger committed:
fixed image error
fixed customize error

matthias-luger committed:
customize flix

matthias-luger committed:
ported the profitable crafts changes to next

matthias-luger committed:
Merge remote-tracking branch 'origin/develop' into next

matthias-luger committed:
merge

matthias-luger committed:
Merge branch 'develop' into next

matthias-luger committed:
fixes

Flou21 committed:
update dockerfile for next

Flou21 committed:
npm ci in dockerfile

matthias-luger committed:
fixed links

matthias-luger committed:
Merge branch 'next' of https://github.com/Coflnet/hypixel-react into next

matthias-luger committed:
fixed auction details link

matthias-luger committed:
added embeds
improved multiple things

matthias-luger committed:
- updated material icon package
- improved lighthouse score
- removed unused index.html

matthias-luger committed:
removing autonumeric to decrease bundle size

matthias-luger committed:
improved SSR
improved SEO
changed rendering of virtualized lists for SSR

matthias-luger committed:
removed wrong service worker
removed onerror handler

matthias-luger committed:
changed image ratios

matthias-luger committed:
added width back for coin.png

matthias-luger committed:
readded auction details in flipper

matthias-luger committed:
fixed flipper styling
fixed filter update
fixed item notifier
ssr optimised

matthias-luger committed:
fixed not loading of auction details
fixed falsely rendered navbar
improved error page stylings
improved auction embed

matthias-luger committed:
fixed search bug

Flou21 committed:
update docker files

Ekwav commented on LICENSE:
```diff
@@ -1,661 +0,0 @@
-                    GNU AFFERO GENERAL PUBLIC LICENSE
```
Why was the licence removed?

matthias-luger committed:
fixed wrong loading of the flipper
fixed search bar for blacklist
fixed some next warnings
fixed loading of auction details

matthias-luger committed:
Merge branch 'next' of https://github.com/Coflnet/hypixel-react into next

matthias-luger committed:
removed console.log

matthias-luger committed:
added redirects
added filter to the embeds
fixed styling loading issue

matthias-luger committed:
removed debugger

matthias-luger committed:
fixed error if no search results are returned
